src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -99,14 +99,20 @@ private function addRoute(RouteCollection $routeCollection, string $resourceClas
             throw new RuntimeException('Either a "route_name" or a "method" operation attribute must exist.');
         }
 
+        $actionName = sprintf('%s_%s', strtolower($operation['method']), $collection ? 'collection' : 'item');
+        $controller = self::DEFAULT_ACTION_PATTERN.$actionName;
+
         if (isset($operation['controller'])) {
             $controller = $operation['controller'];
-        } else {
-            $actionName = sprintf('%s_%s', strtolower($operation['method']), $collection ? 'collection' : 'item');
-            $controller = self::DEFAULT_ACTION_PATTERN.$actionName;
+            $actionName .= sprintf('_%s', $operation['name'] ?? $controller);
```

Comment:
Sorry, I don't see how it's related. For operations declared via annotations, there has always been a name, just like the defaults.